webapi/appsettings.json

```diff
@@ -15,11 +15,13 @@
   // - Optionally set TimeoutLimitInS to the maximum number of seconds to wait for a response from the AI service. If this is not set, there is no timeout.
   // - Optionally set SemanticSkillsDirectory to the directory from which to load semantic skills (e.g., "./SemanticSkills").
   // - Optionally set KeyVaultUri to the URI of the Key Vault for secrets (e.g., "https://contoso.vault.azure.net/").
+  // - Optionally set InMaintenance to true to set the application to maintenance mode.
   //
   "Service": {
     // "TimeoutLimitInS": "120"
     // "SemanticSkillsDirectory": "",
     // "KeyVault": ""
+    // "InMaintenance":  true
```

Comment:
If this is set to true, how does the app come out of maintenance mode without redeploying it?

Reply:
Set to false.  Using the config like this would be for something like the partition upgrade (external to the api service).

With the memory-migration, it will dynamically indicate maintenance as an overlay to this config (or condition)

Reply:
Discussed offline, I get it now, thanks!